src/Symfony/Bridge/Twig/Tests/Extension/FormExtensionBootstrap3LayoutTest.php

```diff
@@ -35,7 +35,7 @@ protected function setUp()
             'bootstrap_3_layout.html.twig',
             'custom_widgets.html.twig',
         ));
-        $renderer = new TwigRenderer($rendererEngine, $this->getMock('Symfony\Component\Form\Extension\Csrf\CsrfProvider\CsrfProviderInterface'));
+        $renderer = new TwigRenderer($rendererEngine, $this->getMock('Symfony\Component\Security\Csrf\CsrfTokenManagerInterface'));
```

Comment:
The version is not bumped because of this change, but is required because of the new abstract base test class which was introduced in #12948 (see the error message @Tobion posted in one of his comments).